.github/workflows/publish.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           mv types/* "$GITHUB_WORKSPACE/v${{ steps.version_check.outputs.version }}/types"
 
           mkdir -p "$GITHUB_WORKSPACE/v${{ steps.version_check.outputs.version }}/examples"
-          mv examples/*.json "$GITHUB_WORKSPACE/v${{ steps.version_check.outputs.version }}/examples"
+          cp -r examples/**/*.json "$GITHUB_WORKSPACE/v${{ steps.version_check.outputs.version }}/examples"
```

Comment:
`cp` should preserve directory structure (eg subfolders) whereas `mv` would exclusively move/copy `.json` files